packages/webpack-plugin/package.json

```diff
@@ -52,6 +52,7 @@
     "@types/webpack": "^5.28.5",
     "jest": "^29.7.0",
     "msw": "^2.1.5",
+    "codecovProdRollupPlugin": "npm:@codecov/rollup-plugin@0.0.1-beta.2",
```

Comment:
we need a todo to add dependabot for this so we can bump right after we do a release.